README.md

```diff
@@ -2,6 +2,46 @@
 
 Simple list of top-level packages in Python's stdlib
 
+Note: If you only need the live module names on 3.10+, just use
+`sys.stdlib_module_names`.  This is not exactly a backport, but a static list of
+those for most useful Python versions.
+
+If someone wants to add alternate runtimes, PRs welcome.
+
+
+Usage
+-----
+
+```pycon
+>>> import stdlibs
+```
+
+For common usage, you can use `module_names` which is all the top-level names
+valid in some version of Python 3.  Not necesssarily valid in all of them.
+
+```pycon
+>>> print("os" in stdlibs.module_names)
+True
+>>> print("os.path" in stdlibs.module_names)
+ValueError("os.path has a dot and is not a top-level name")
+```
+
+For more advanced usage, you can pass a version like `"3.6"` or the special
+values `"all"`, `"all2"`, or `"all3"` (that last one is the one used for
+`module_names` above), and treat the result as a set.
+
+```pycon
+>>> old_ver = stdlibs.stdlib_module_names("3.6")
+>>> "dataclasses" in old_ver
+False
+>>> less_old_ver = stdlibs.stdlib_module_names("3.7")
+>>> "dataclasses" in old_ver
+True
+>>> [v for v in stdlibs.KNOWN_VERSIONS
+...  if "dataclasses" in stdlibs.stdlib_module_names(v)]
+['3.7', '3.8', '3.9', '3.10']
+```
```

Comment:
This changed with the move to modules. Maybe suggest importing/using the py3x submodules directly unless alongside version string function?